packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.js

```diff
@@ -75,7 +75,7 @@ function SingleImageViewer({
           <title id={title.id}>{title.text}</title>
         )}
         <svg
-          style={{ touchAction: 'none' }}
+          style={{ touchAction: 'pinch-zoom' }}
```

Comment:
I think this is needed to allow for zoom on tablets and phones, but I'm not 100% sure. PFE varies the style according to whether you are drawing or not:
https://github.com/zooniverse/Panoptes-Front-End/blob/85b7a04618b8d0c8621e83366728efde20fc243d/app/classifier/annotation-renderer/svg.jsx#L136-L141

Reply:
`@visx/zoom` also supports pinch gestures, via `zoom.containerRef` passed into the zoom render function.